src/components/RecipePage/RecipePage.js

```diff
@@ -15,9 +15,9 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
-export default function RecipePage(props) {
+export default function RecipePage(props, {isLoading}) {
```

Comment:
This is not the best way to pass down props.
You can either pass the whole `props` object:
i.e. `... RecipiePage(props)` and then grab your props one by one
like `props.RecipieArray`, `props.isLoading`

or you can deconstruct your props object:
i.e. `...RecipePage({ description, name, isLoading, ... })`
and use them directly in your code.

We have done many other times, when in doubt please check your older projects or other components: